src/assets/styles/global.scss

```diff
@@ -1,22 +1,36 @@
 @import "./node_modules/@esri/calcite-colors/colors.scss";
 @import "./node_modules/@esri/calcite-base/dist/_index.scss";
-@import "../../scss/global";
 
 :root {
   font-family: $avenir-family;
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  --calcite-border-radius: 3px;
   @include calcite-theme-light();
+  @include componentLayoutVars();
+  @include componentShadowVars();
+  --calcite-border-radius: 3px;
+  --calcite-ui-opacity-disabled: 0.3;
+
+  --calcite-ui-text-weight: 400;
+  --calcite-ui-text-weight-demi: 600;
+  --calcite-font-family: #{$avenir-family};
+  --calcite-code-family: #{$code-family};
+
+  --calcite-ui-foreground-current: #{$h-bb-010};
```

Comment:
Thinking these two color vars could make there way into base.

Reply:
Where are these used? Thoughts cc @bstifle

Reply:
@macandcheese They're used mostly in Action, but they need to respond to the `theme` prop meaning they kinna gotta live at this level.

Reply:
Yeah, I mean that makes sense, if they were in base they could just come from those mixins. Just a UI treatment we don't have anywhere at this point so would want to make sure they are used elsewhere if we did that.

Reply:
Totes.

Reply:
As a note, the light version is based on what we do on the Content page. 
![image](https://user-images.githubusercontent.com/12503298/90576439-151e0900-e173-11ea-8bb7-e7edc8ae67ab.png)

The dark version is a little different though:
![image](https://user-images.githubusercontent.com/12503298/90576459-26671580-e173-11ea-9fb6-619eec332224.png)

Reply:
@bstifle For reference, the text color on the dark theme is 
`$v-bb-120: #59d6ff`